cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -2723,4 +2725,24 @@ String getInternalServiceHostname(String serviceName)    {
     private Date dateSupplier() {
         return new Date();
     }
+
+    /**
+     * @param current Previous ConfigMap
```

Comment:
The parameter is current but the description says Previous, it sounds confusing :)